src/main/java/teammates/ui/template/InstructorStudentsListStudentsTableCourses.java

```diff
@@ -0,0 +1,56 @@
+package teammates.ui.template;
+
+import teammates.common.util.Sanitizer;
+
+public class InstructorStudentsListStudentsTableCourses {
+
+    private boolean isCourseArchived;
+    private String courseId;
+    private String courseName;
+    private String googleId;
+    private String numStudents;
+    private String instructorCourseEnrollLink;
+    private boolean isInstructorAllowedToModify;
+
+    public InstructorStudentsListStudentsTableCourses(boolean isCourseArchived, String courseId, String courseName,
+                                                     String googleId, String numStudents,
+                                                     String instructorCourseEnrollLink,
+                                                     boolean isInstructorAllowedToModify) {
+        this.isCourseArchived = isCourseArchived;
+        this.courseId = courseId;
+        this.courseName = Sanitizer.sanitizeForHtml(courseName);
+        this.googleId = googleId;
+        this.numStudents = numStudents;
+        this.instructorCourseEnrollLink = instructorCourseEnrollLink;
+        this.isInstructorAllowedToModify = isInstructorAllowedToModify;
+    }
+    
+    public boolean getIsCourseArchived() {
+        return isCourseArchived;
+    }
+
+    public String getCourseId() {
+        return courseId;
+    }
+
+    public String getCourseName() {
+        return courseName;
+    }
+
+    public String getGoogleId() {
+        return googleId;
+    }
+
+    public String getNumStudents() {
+        return numStudents;
+    }
+
+    public String getInstructorCourseEnrollLink() {
+        return instructorCourseEnrollLink;
+    }
+
+    public boolean getIsInstructorAllowedToModify() {
+        return isInstructorAllowedToModify;
+    }
```

Comment:
Well, I don't want to name it `getIs___` either, circumstances (read: JSTL's requirement) forced me to name it that way.
I guess removing the 'is' part works.